src/kw_db.sh

```diff
@@ -0,0 +1,83 @@
+include "$KW_LIB_DIR/kwio.sh"
+include "$KW_LIB_DIR/kw_date_and_time.sh"
+
+DB_NAME='kw.db'
+DB_PATH="$KW_ETC_DIR/db_files/$DB_NAME"
+
+function create_table()
+{
+  local table="$1"
+  local entries="$2"
+  local db="${3:-$DB_PATH}"
+
+  sqlite3 "$db" -batch "CREATE TABLE $table ($entries);"
+}
+
+function insert_into()
+{
+  local table="$1"
+  local entries="$2"
+  local values="$3"
+  local db="${4:-$DB_PATH}"
+
+  sqlite3 "$db" -batch "INSERT INTO $table ($entries) VALUES ($values);"
+}
+
+function show_table_contents()
+{
+  local table="$1"
+  local contents="${2:-'*'}"
+  local db="${3:-$DB_PATH}"
+
+  sqlite3 "$db" -batch "SELECT $contents FROM $table"
+}
+
+function format_entries()
+{
+  local entries=''
+
+  for entry in "$@"; do
+    entry="${entry//\'/\'\'}" # escapes single quotes
+    entries+="'$entry',"
+  done
+
+  printf '%s\n' "${entries%?}" # removes last comma
+}
+
+function read_sql()
+{
+  local sql_path="$1"
+  local db="${2:-$DB_PATH}"
+
+  if [[ ! -f "$sql_path" ]]; then
+    printf '%s\n' "Could not find the file: $sql_path"
+    return 22
+  fi
+
+  sqlite3 "$db" < "$sql_path"
+}
```

Comment:
Commit 1: Add support to create a database by using an SQL file.
1. Find a better name for this function
2. Validate each parameter
3. Add documentation
4. Add unit test